lib/Config.js

```diff
@@ -193,7 +193,7 @@ class Config {
         const dataPath = process.env.S3DATAPATH ?
             process.env.S3DATAPATH : 'localData';
```

Comment:
Let's be consistent and set this one with `${_dirname}/../localData` like we are doing now for metadata.